content/en/docs/setup/install/istioctl/test.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+# shellcheck disable=SC2034,SC2153,SC2155,SC2164
+
+# Copyright Istio Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+#source "tests/util/samples.sh"
+
+# Install Istio
+# @setup profile=none
+snip_install_istio_using_the_default_profile_1
+_wait_for_deployment istio-system istiod
+
+# ?? not sure how could I test this and the following yaml?
+#snip_install_istio_using_the_default_profile_2
+#_wait_for_deployment istio-system istiod
+
+# ?? also how could i test manifest
+
+#
+
+snip_install_a_different_profile_1
+_verify_like snip_check_whats_installed_1 snip_check_whats_installed_1_out
```

Comment:
Fix bad variable syntax, here and throughout the test.
```suggestion
_verify_like snip_check_whats_installed_1 "$snip_check_whats_installed_1_out"
```